Loop/Extensions/DeviceDataManager+BolusEntryViewModelDelegate.swift

```diff
@@ -45,11 +45,11 @@ extension DeviceDataManager: BolusEntryViewModelDelegate {
     
     var isGlucoseDataStale: Bool {
         guard let latestGlucose = glucoseStore.latestGlucose else { return true }
-        return Date().timeIntervalSince(latestGlucose.startDate) <= loopManager.settings.inputDataRecencyInterval
+        return -latestGlucose.startDate.timeIntervalSinceNow > loopManager.settings.inputDataRecencyInterval
```

Comment:
[nit] not sure adding an additional operator (the negation operation) is really clearing things up here. Now you still have to understand that `timeIntervalSince` is typically negative for this type of comparison *and* apply the negative operator and then evaluate the comparison operation. This seems like a step backwards.

Reply:
Come to think of it, this small amount of logic is not covered by unit tests (otherwise I would have found this bug!).  I think I will fix that.